_component_design/checkboxes.md

```diff
@@ -18,25 +18,90 @@ helix-ui-javascript: true
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use checkboxes in the following situations:
+
+- When the user needs to answer a question or wants to change a control panel setting. Do not use a checkbox to trigger another process or workflow.
+- When the user can select any number of choices, including zero, one, or several choices. If a user can only select one option, use a radio button or an input selector.
+- When the user can turn on or turn off a single option. For example, use a single checkbox when a user accepts the terms of an End User License Agreement (EULA).
+- When it is important for the user to compare all options simultaneously. If it is not important for the user to compare all options at the same time, consider using an input selector. An input selector uses less space than a checkbox group (but introduces greater cognitive load), and is more difficult to navigate for users who have difficulty making precise mouse movements.
+
```

Comment:
add a when to use about when a user needs to select a row in a table to act of that item.